pkg/storage/segments/store.go

```diff
@@ -418,3 +417,35 @@ func convertTime(ts *timestamp.Timestamp) time.Time {
 	}
 	return t
 }
+
+func split(path storj.Path) (projectID, bucket string, objectPath storj.Path, segmentIndex int64, err error) {
+	comp := storj.SplitPath(path)
+	if len(comp) < 3 {
```

Comment:
It is ok for accessing only bucket? Won't it be `l/bucket_name` without additional path? Also I'm not sure if `projectID` will be visible on client side.